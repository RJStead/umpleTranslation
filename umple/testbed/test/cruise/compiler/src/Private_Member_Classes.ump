generate Java "../src-gen-umple";

class Class4 {
 Integer id;

}

class Class3
{
    Class4 c4;
    public int foo() {
        if (c4 != null) {
            return c4.id;
        }
        return -1; 
    }
}
