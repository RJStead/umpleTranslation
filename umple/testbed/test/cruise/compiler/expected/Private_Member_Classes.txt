Private_Member_Classes.ump:13: error: id has private access in Class4
            return c4.id;
                     ^
1 error